index.html

```diff
@@ -34,5 +34,154 @@ <h2 class="welcoming">Hey there. I’m Anas
          </ul>
        <div class="angle"> <img src="images/angle.png" alt=""></div> 
    </section>
+
+   <section class="works">
+   
+   <h3 class="Section-title">My Recent Works</h3>
+      <div class="card card1">
+        <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
```

Comment:
- Please use the appropriate background for this language as it is on the Figma design ` #EBF0EE;` but you used the wrong one on your design `#eee0ee;`. Please change it as appropriate.

##### Figma Design
![Screenshot from 2022-06-01 17-43-49](https://user-images.githubusercontent.com/58771507/171457112-f15e396f-14e6-482b-aea8-48072460aef9.png)

##### Your Design
![Screenshot from 2022-06-01 17-44-18](https://user-images.githubusercontent.com/58771507/171457209-549ca389-f8de-4df0-8a27-284c6ce35563.png)